src/test/java/com/github/tomakehurst/wiremock/StubbingAcceptanceTest.java

```diff
@@ -1221,6 +1224,31 @@ void queryParamCanBeMatchedAsNotAbsent() {
     assertThat(testClient.get("/search").statusCode(), is(404));
   }
 
+  @Test
+  void cannotCreateMultipleStubMappingsWithTheSameId() {
```

Comment:
Is it also worth adding a coresponding test to the `AdminApiTest` to show the errors we get in the admin API when we use a duplicate id ?

Reply:
Yes, I think that's a good idea. This test already makes its create request via the admin HTTP client, so is basically already making its assertions on the errors returned by the admin API, but if you think a more explicit test would be worth it, I will add it.